frontend/src/scenes/insights/insightLogic.ts

```diff
@@ -11,6 +12,32 @@ export const ViewType = {
     RETENTION: 'RETENTION',
     PATHS: 'PATHS',
 }
+interface FilterType {
+    insight: 'TRENDS' | 'SESSIONS' | 'FUNNELS' | 'RETENTION' | 'PATHS' | 'LIFECYCLE' | 'STICKINESS'
+    display?:
+        | 'ActionsLineGraph'
+        | 'ActionsLineGraphCumulative'
+        | 'ActionsTable'
+        | 'ActionsPie'
+        | 'ActionsBar'
+        | 'PathsViz'
+        | 'FunnelViz'
+    interval?: string
+    date_from?: string
+    date_to?: string
+    properties?: Record<string, any>[]
```

Comment:
This type already exists: PropertyFilter from types.ts

So this should be `properties: Array<PropertyFilter>`